foreman_maintain.gemspec

```diff
@@ -18,8 +18,7 @@ the Foreman/Satellite up and running."
   s.files += `git ls-files extras`.split("\n")
   s.extra_rdoc_files = ['LICENSE', 'README.md']
   s.require_paths = ['lib']
-  s.executables = ['foreman-maintain',
-                   'foreman-maintain-complete', 'foreman-maintain-rotate-tar']
```

Comment:
Can we just drop the whole script at this point?

Reply:
Removing it from `s.executables` ensures that the gem-installer does not create a ruby-wrapper script in `/usr/bin/foreman-maintain-rotate-tar`.

I am not fully sure, why tar needs to use the shell-script when changing volume.
Also it is only set, if the tar-command does not use `--gzip`.
The history says we need it for the extract to work smoothly: 2beadc61aefc5caff5813198520d48027eb13baa